KarhooUISDK/CommonUI/Checkbox/CheckboxView.swift

```diff
@@ -0,0 +1,155 @@
+//
+//  CheckboxView.swift
+//  KarhooUISDK
+//
+//  Created by Aleksander Wedrychowski on 04/02/2022.
+//  Copyright © 2022 Flit Technologies Ltd. All rights reserved.
+//
+
+import UIKit
+
+class CheckboxView: UIControl {
+
+    // MARK: - Nested types
+
+    // Some ui properties are not default, so need to describe it properly. This is because visible view is smaller than proper UX expects.
+    enum CustomConstants {
+        static let visibleIconSideLenght: CGFloat = 18
+        static let sideLenght: CGFloat = UIConstants.Dimension.Button.small
+        static let visibleAndActualWidthOffset: CGFloat = sideLenght - visibleIconSideLenght
+    }
+
+    private enum SelectionState {
+        case selected
+        case unselected
+
+        var image: UIImage {
+            switch self {
+            case .selected: return .uisdkImage("checkbox_selected")
+            case .unselected: return .uisdkImage("checkbox_unselected")
+            }
+        }
```

Comment:
checkbox could be done as view with background, border and only tick mark as an image. Then we can change any component color programmatically, without any changes in svg file. It will make easier to add dark mode in future to this component.